### ISSUE TITLE
Cookies not Saved

### ISSUE THREAD

**Issue by Caesar-DE** [codebar-ag/laravel-docuware#48]
Hello,
Wenn I set the "DOCUWARE_COOKIES"  in the .env file, it gives an Error when I use the method " DocuWare::uploadDocument".
Error Message:
You are not authorized. Make sure you are logged in with the correct credentials. Try to use "->login()" before you make other requests.

I had used the Method "DocuWare::login()" but this had not helped.

And when I unset the "DOCUWARE_COOKIES" , then can I used all Methods but after 3 or 4 uploads it gives "None of the registered licenses are available for client XXXXX" .
Basically,   the number of licenses is exceeded

**Comment by StanBarrows** [comment]
hey @zaidan1 , thanks for your message.

Lets try the following:

- [ ] Uncomment or remove the DOCUWARE_COOKIES within the .env file.  Just keeping it empty does not work.

```
#DOCUWARE_COOKIES=""
```

- [ ] Run the DocuWare::login() method within your application
- [ ] Run the following Command to receive the correct cookie

```
<?php

namespace App\Console\Commands\DocuWare;

use Illuminate\Console\Command;
use Illuminate\Support\Facades\Cache;

class ListCookie extends Command
{
    /**
     * The name and signature of the console command.
     *
     * @var string
     */
    protected $signature = 'docuware:list-cookie';

    /**
     * The console command description.
     *
     * @var string
     */
    protected $description = 'List DocuWare Cookie';

    /**
     * Execute the console command.
     *
     * @return int
     */
    public function handle()
    {
        $cacheKey = 'docuware.cookies';
        $cookie = Cache::get($cacheKey);
        if (!$cookie) {
            $this->info('No cookie found for the Key "' . $cacheKey . '".');
        }
        if ($cookie) {
            $this->info(implode($cookie));
        }

        return Command::SUCCESS;
    }
}
```

- [ ] Enable DOCUWARE_COOKIES within your .env and aste the output from the command in there.

```
DOCUWARE_COOKIES="$output"
```
